Sources/MapboxDirections/RouteOptions.swift

```diff
@@ -122,6 +126,13 @@ open class RouteOptions: DirectionsOptions {
      */
     open var roadClassesToAvoid: RoadClasses = []
     
+    /**
+     The route classes that the calculated routes will allow.
+     
+     This property has no effect unless the profile identifier is set to `DirectionsProfileIdentifier.automobile` or `DirectionsProfileIdentifier.automobileAvoidingTraffic`.
+    */
+    open var roadClassesToAllow: RoadClasses = []
```

Comment:
With this name, the developer may assume this property must contain a road class in order for the route to traverse roads of that class, even if in reality the routing profile would include it by default. If the developer redundantly sets this property to a value such as `RoadClasses.motorway`, will the Directions API silently ignore the redundant value, or will it return an error? If the API only allows certain values, then we should, split the enumeration in two, special-case and ignore the allowed-by-default classes, or document which cases are allowed with which property. None of the approaches is particularly attractive, but perhaps a future Directions API enhancement could allow us to design a more ergonomic, predictable client-side API.

Reply:
😞 I will fix this one. I will check how the directions api behaves and make a decision from there.